Automattic/wp-calypso
# Include Powered by Titan logo on email comparison page

#### Changes proposed in this Pull Request

* This PR adds the "Powered by Titan" logo to the email comparison page

#### Testing instructions

* Ensure you are running with the `titan/phase-2` feature flag enabled.
* Go to Manage -> Domains, and click on the "Add" button for a domain you own.
* Verify that the "Powered by Titan" logo is present and looks good.

<img width="1114" alt="email_comparison_powered_by_titan_i2" src="https://user-images.githubusercontent.com/3376401/105481542-e242e880-5caf-11eb-995e-8ef6b31c43c3.png">
